vars/piperPipelineStageSecurity.groovy

```diff
@@ -41,60 +42,55 @@ void call(Map parameters = [:]) {
         .addIfEmpty('whitesourceExecuteScan', script.commonPipelineEnvironment.configuration.runStep?.get(stageName)?.whitesourceExecuteScan)
         .use()
 
```

Comment:
we should probably keep the outer `piperStageWrapper`.
This way the extension security stage cannot be overwritten unless one of the steps is active.

Reply:
good point!